pkg/curatedpackages/config/awssecret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+data:
+  ID: {{.eksaAccessKeyId}}
```

Comment:
This is basically like hardcoding credentials. I know this was discussed in the design doc, but have we considered making IRSA a requirement for this and leveraging that so we don't need a whole user for pull the images?

Reply:
@junshun might have a good answer on this. Thoughts?

Reply:
For now we are currently suggesting that they use this method. IRSA could work however we won't be suggesting. IRSA also makes it more difficult as we migrate towards the long term solution. Since we are suggesting that the customer will create a role with limited scope (just pulling images/getting token) there should be minimal risks.

Reply:
I understand, but there is a difference between creating a role and creating a user, and here it is specifying creating a user. But if IRSA makes it more difficult maybe this is an okay step in that direction.